kolibri_explore_plugin/kolibri_plugin.py

```diff
@@ -54,6 +54,7 @@ def plugin_data(self):
             "initialContentPack": conf.OPTIONS["Explore"][
                 "INITIAL_CONTENT_PACK"
             ],
+            "hideDiscoveryTab": conf.OPTIONS["Explore"]["HIDE_DISCOVERY_TAB"],
```

Comment:
I have no objections, this seems fine, but I wonder about using either a blocklist or an allowlist for the tabs that should be hidden or shown. That way this can be fine-tuned more easily in the future, for more cases.

Reply:
Interesting idea. I asked Design (Joana) about it and there aren't more tabs planned in the future. Quite the opposite: ideally we should use Kolibri's Library page in the future instead of our current fork. So a blocklist would be a good solution for generalizing the logic but at this point it will be against the KISS principle. Anyways thanks so much for your feedback and thinking about potential improvements!